deployment/api_transparency_dev/main.tf

```diff
@@ -143,11 +143,11 @@ resource "google_compute_url_map" "default" {
     }
     path_rule {
       paths = [
-        "/distributor-ci/*"
+        "/ci/*"
```

Comment:
Will this expose metrics etc. too?

If so, could this be "/ci/distributor/*" and then `path_prefix_rewrite` be `/distributor/` below?

Reply:
Good call.